src/parser.ml

```diff
@@ -326,8 +330,9 @@ and parser env = "[" t:(term PAppl) ts:{"," (term PAppl)}* "]" -> t::ts
 
 (** [arg] parses a single function argument. *)
 and parser arg =
-  | x:ident                            -> (x, None   )
-  | "(" x:ident ":" a:(term PFunc) ")" -> (x, Some(a))
+  | x:ident                            -> (x, None,    false)
```

Comment:
There should be a case for "{" x:ident "}" too.

Reply:
Indeed, thanks !